### PR TITLE
Stack trace removal from UI & REST API Response

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Removed stack tracing metric in KuraPayload.
Removed stack trace panel from Web UI.

### CODE REVIEW

File: kura/org.eclipse.kura.core.cloud/src/main/java/org/eclipse/kura/core/cloud/MessageHandlerCallable.java [hunk 1]
```diff
@@ -236,11 +236,11 @@ public KuraMessage setResponseCode(KuraPayload payload, int responseCode) {
     public void setException(KuraPayload payload, Throwable t) {
         if (t != null) {
             payload.addMetric(METRIC_EXCEPTION_MSG, t.getMessage());
-            payload.addMetric(METRIC_EXCEPTION_STACK, stackTraceAsString(t));
         }
     }
 
-    private String stackTraceAsString(Throwable t) {
+    @SuppressWarnings("unused")
+	private String stackTraceAsString(Throwable t) {
```
Comment (MMaiero) [inline kura/org.eclipse.kura.core.cloud/src/main/java/org/eclipse/kura/core/cloud/MessageHandlerCallable.java]: Please check the formatting. As a reference, please use the cleanup and formatting profiles available here: https://github.com/eclipse/kura/tree/develop/kura/setups/formatting

**Comment by MMaiero** [comment]
Probably the modal window needs to be reshaped a bit, since we don't have anymore the stack trace, right?

Screenshot 2021-05-12 at 13 38 04

**Comment by salvatore-coppola** [comment]
Yes, the stack trace is gone. How should we reshape ? Resizing the box ?

**Comment by MMaiero** [comment]
Yes, I would review the modal to have a more tight fit

**Comment by salvatore-coppola** [comment]
screenshot-modal-error

**Review by MMaiero — APPROVED** [review]

**Pull request merged**